### ISSUE TITLE
Get alternate image sizes from product?

### ISSUE THREAD

**Issue by rbsmidt** [woothemes/woocommerce-rest-api-docs#23]
Is it possible to get other image sizes than just the original from the REST API?
As i can figure out, the only way would be to get media ID and make another request using the WP API to wp-json/wp/v2/media/<id> endpoint.
But that seems like an awful lot of get requests, if you're looping through products from woo rest api, and just want to present a smaller image than the uploaded to the user.

So to sum it up - am i missing some obvious way to solve this?

Best regards

**Comment by claudiosmweb** [comment]
Here is not a support channel.
Please post only issues with our documentation.

**Comment by rbsmidt** [comment]
ahh, my bad @claudiosmweb - sorry for the inconvenience! I've posted my question on SO instead. Thanks for clarifying.

**Comment by steffanhalv** [comment]
@rbsmidt link please ?
